apis/validator/attestation_data.yaml

```diff
@@ -4,8 +4,10 @@ get:
     - Validator
   operationId: "produceAttestationData"
   summary: "Produce an attestation data"
+  deprecated: true
```

Comment:
this should be on the `committee_index` query parameter

Reply:
Should it? This is basically @mcdee's idea, I think, to deprecate the parameter, and I already articulated considerations regarding that.

I don't actually think the endpoint should be kept with the same name but with one fewer parameter, so I'm not sure that it makes sense to deprecate a parameter as such.

I'm not strongly opposed, just so long as it's clear that it still has to be passed to the `produceAttestationData` endpoint regardless of how useless it is in Electra, but that's not how I'd interpret a per-parameter deprecation to begin with.

Reply:
Given that some beacon nodes currently require the parameter we can't remove it, but we can deprecate it so that it is clear that its value is ignored by the beacon node receiving the request.  Then over the lifespan of electra beacon nodes can stop making the parameter required, move to ignoring the parameter entirely, and then it can be removed from the spec at fulu with no ill effects.

But yes, I think the point here is that the tag appears to be in the incorrect place in the doc.

Reply:
I think the approach outlined by Jim makes the most sense, if we wanted to go with a v2 api that removes the query parameters already now we should have done that as part of the attestation api v2 changes but personally I didn't feel strongly about doing that because technically it's not required, the beacon node can just ignore the `committee_index` in the query parameter which is what we / most bns do already.

I think deprecating the `committee_index` now and removing it in Fulu will probably be fine but I am still wondering if it's worth to do this clean up at all, seems like a nit to me